src/utils/feature-flags.ts

```diff
@@ -16,6 +16,9 @@ export const FF_DEV_1284 = 'fflag_fix_front_dev_1284_auto_detect_undo_281022_sho
 // Fix crosshair working with zoom & rotation
 export const FF_DEV_1285 = 'ff_front_dev_1285_crosshair_wrong_zoom_140122_short';
 
+// Add visibleWhen="choice-unselected" option
+export const FF_DEV_1372 = 'ff_front_dev_1372_visible_when_choice_unselected_11022022_short';
```

Comment:
I see the FF is being declared here but its not being used in this PR - was this added by accident?